tasks/sep/fp-recovery/005-set-game-implementation/justfile

```diff
@@ -0,0 +1,28 @@
+set positional-arguments
+
+# default recipe to display help information
+default:
+  @just --list
+
+# Generate the `input.json` with the game type and implementation to set.
+generate-input gameType='' implAddr='':
+  #!/usr/bin/env bash
+  set -euo pipefail
+  . .env
+  SET_GAME_TYPE_SIG="setImplementation(uint32,address)"
+  ENCODED_CALL=$(cast calldata "${SET_GAME_TYPE_SIG}" "{{gameType}}" "{{implAddr}}")
+
+  CONFIG_URL="https://raw.githubusercontent.com/ethereum-optimism/superchain-registry/refs/heads/main/superchain/configs/${L1_CHAIN_NAME}/${L2_CHAIN_NAME}.toml"
+  CONFIG_TOML=$(curl -s --show-error --fail "${CONFIG_URL}")
+  SC_CONFIG_ADDR=$(echo "${CONFIG_TOML}" | yq -p toml .addresses.SystemConfigProxy)
+  PROXY_ADMIN_OWNER_ADDR=$(echo "${CONFIG_TOML}" | yq -p toml .addresses.ProxyAdminOwner)
+
+  DGF_ADDR=$(cast call "${SC_CONFIG_ADDR}" "disputeGameFactory()(address)")
+  echo "OWNER_SAFE=${PROXY_ADMIN_OWNER_ADDR}" >> .env
```

Comment:
NIT: When we launch multiples time the script this will append into the `.env` the `OWNER_SAFE`.
This shouldn't have any impact, I was thinking of a case when we run the script for the first time and wants to know how it works. 
By adding multiples times this parameters this shouldn't not have impact, should we still fix this by checking if the `OWNER_SAFE` is already written into the `.env` or this is overkill in your opinion? 
here -> https://github.com/ethereum-optimism/superchain-ops/blob/b73271f4bcb636f5a34a1857b071dce8f8fff4a5/tasks/sep/fp-recovery/005-set-game-implementation/justfile#L21


![CleanShot 2024-11-01 at 14 43 33@2x](https://github.com/user-attachments/assets/a7e0f6c5-ee0f-4bce-a2ac-d6fb61fee31f)

Reply:
I figured it wasn't worth it given the last value is the one that gets used anyway.  The smarter approach to this in https://github.com/ethereum-optimism/superchain-ops/pull/363 makes it much less likely this will happen as there's a separate `prep` task and then multiple set-implementation calls can be added to the batch.